javascript/react/src/components/ArqueroDetailsList/renderers/StatsColumnHeader.tsx

```diff
@@ -32,7 +33,11 @@ export const StatsColumnHeader: React.FC<RichHeaderProps> = memo(
 				// data type is on the field, not the meta...
 				const value: any = stat === StatsColumnType.Type ? field.type : st[stat]
 				return (
-					<StatCell name={stat} value={value} key={`${column.key}-${stat}`} />
+					<StatCell
+						name={stat}
+						value={value}
+						key={`${column?.key || field.name}-${stat}`}
```

Comment:
IColumn.key should always be required, but since we're reusing this in a new context it isn't always there. This is a good indication to split this out into a better base component (and probably the histogram header too).

Reply:
I'll create another PR for this change as we talked about